.pylintrc

```diff
@@ -151,6 +151,8 @@ disable=print-statement,
         wrong-import-position,
         too-few-public-methods,
         ungrouped-imports, 
+        fixme, 
+        too-many-function-args, 
```

Comment:
Consider using 
`#pylint: disable=unused-variable`
on the function where you need more.